proto/v1/setting_service.proto

```diff
@@ -469,3 +470,8 @@ enum DatabaseChangeMode {
   // A simple database change process in SQL editor. Users can execute SQL directly.
   EDITOR = 2;
 }
+
+message MaximumSQLResultSizeSetting {
```

Comment:
Also comment about the default value (100MB).

Reply:
I think we don't need to hack the 0, the value should larger than 0, and just insert the 100 as initial setting by default